### ISSUE TITLE
hovering certain code parts yield "Request textDocument/hover failed."

### ISSUE THREAD

**Issue by apollo13** [microsoft/pyright#7080]
**Describe the bug**
Hovering (at least at this exact commit):
https://github.com/apollo13/community.general/blob/fffe2950dce5d54e92c87602820120c451c4c81c/plugins/modules/consul_auth_method.py#L134
yields:
```
[Error - 6:14:46 PM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Debug Failure. False expression.
  Code: -32603 
```

As with https://github.com/microsoft/pyright/issues/6981, details are sparse but I will try my best. I think this should be reproducible by checking out https://github.com/apollo13/community.general and switching to commit `fffe2950dce5d54e92c87602820120c451c4c81c`. Make sure that the checkout is in a folder structure like this:
`/home/florian/sources/ansible/ansible_collections/community/general` and then set the following as workspace settings:
```
{
    "python.analysis.extraPaths": [
        "/home/florian/sources/ansible"
      ],    
}
```
So basically ensure that `ansible_collections.community.general` is resolvable. Happy to help debug this further.

**VS Code extension or command-line**
VS Code extension v1.1.347

**Comment by erictraut** [comment]
I just published pyright 1.1.348. Once the community-maintained pyright wrapper on pypi has been updated (usually within 24 hours), please try to repro your problem with this new version. There's a decent chance that the problem has been fixed. If not, we'll diagnose further.

**Comment by apollo13** [comment]
Indeed, thank you very much!
